fms_extras/utils/generation.py

```diff
@@ -20,6 +634,23 @@ def __right_pad_zeros(input_tensor: torch.Tensor, max_length: int) -> torch.Tens
     )
 
 
+def __right_pad_cache_block_mapping(
+    cache_data: PagedAttentionCacheData, block_mapping_max: int
+):
+    """
+    right pad the block mapping when cudagraphs is enabled
+
+    Args:
+        cache_data: PagedAttentionCacheData
+            the paged-attention cache data
+        block_mapping_max: int
+            the maximum number of blocks per sequence required to complete generation
+    """
+    cache_data.block_mapping = __right_pad_zeros(
+        cache_data.block_mapping, block_mapping_max
```

Comment:
since this is a private function just calling one other private function, should we modify __right_pad_zeros instead? I think that will lead to a minor simplification in paged_generate too.

Reply:
fixed in latest commit